src/components/top-tab-bar/TopTabBar.tsx

```diff
@@ -42,8 +42,14 @@ const createTabBarStyles = (themeColors: ThemeColors) =>
   StyleSheet.create({
     tabBarContainer: {
       flexDirection: 'row',
+      marginBottom: -4,
       paddingBottom: 4,
-      position: 'relative'
+      position: 'relative',
+      zIndex: 100,
```

Comment:
just out of curiosity, do we have defined zIndex layers for our app?

Reply:
We do in the web client, probably should have it for mobile too

Reply:
I don't think we do, but it would be awesome to define some constants for this. Especially on mobile